k8s_handle/templating.py

```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
                 raise TemplateRenderingError('Unable to render {}, due to: {}'.format(template, e))
         return output
 
+    def _filter_tagged_templates(self, templates, only_tags, skip_tags):
+        return [i for i in templates if self._evaluate_tags(i, only_tags, skip_tags)]
```

Comment:
More idiomatic way:
```
return templates.filter(lambda i: self._evaluate_tags(i, only_tags, skip_tags)) 
```

Reply:
So, maybe `def _filter_tagged_templates(self, templates, only_tags, skip_tags)`, not required at all.

Reply:
Ok, I replaced `_filter_tagged_templates()` to `filter()` + `lambda`:
```python
templates = filter(
    lambda i: self._evaluate_tags(self._get_template_tags(i),
                                  settings.ONLY_TAGS,
                                  settings.SKIP_TAGS),
    templates
)
```